packages/sdk/src/cosmwasmclient.ts

```diff
@@ -92,6 +102,39 @@ export interface ContractDetails extends Contract {
   readonly initMsg: object;
 }
 
+/** A transaction that is indexed as part of the transaction history */
+export interface IndexedTx {
+  readonly height: number;
+  readonly hash: string;
```

Comment:
This is upper-case hex, right?

Reply:
Jap, documented in 23f88564e5df7ec092b9caf76708cb26d1cd12bc